plugins/maven/src/main/java/org/jetbrains/idea/maven/project/MavenGeneralSettingsWatcher.kt

```diff
@@ -49,6 +54,16 @@ class MavenGeneralSettingsWatcher private constructor(
     }, parentDisposable)
   }
 
+  private fun updateImportingSettings() {
+    if (generalSettings.isTychoProject) {
+      val importingSettings = manager.importingSettings
+      val dependencyTypes = importingSettings.dependencyTypesAsSet
+      dependencyTypes.add(MavenTychoConstants.PACKAGING_ECLIPSE_PLUGIN)
+      dependencyTypes.add(MavenTychoConstants.PACKAGING_ECLIPSE_TEST_PLUGIN)
```

Comment:
Why not adding the other types (in particular `eclipse-feature` or `eclipse-target-definition`) ?

Reply:
Eclipse Features or Target definitions should not contain any code, they're just grouping things.
Maven does not need to scan and know about them. The real stuff happens for Plugins only.
I may be wrong tho 👀

Reply:
That is true even for other packaging, like `eclipse-repository` but then it's also true for _regular_ maven packaging like `pom` (used for _bill-of-material_ POMs).

I'm don't know how IJ handles that as POMs are not listed in the default _dependency types_ list in maven settings, but it makes sense they should be imported, at least internally ?

Reply:
@bric3 it could make sense, yeah. Main reason I excluded them is because of resolve time. Including all features and other packaging types is probably going to take much more time for the entire workspace to be resolved.